<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreateFigure</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>7</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>430</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title">
            <string/>
           </attribute>
           <widget class="QComboBox" name="comboBox">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>151</width>
              <height>21</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Фигура не выбрана</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Точка</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Прямая</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Куб</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Параллелепипед</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Пирамида</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Сфера</string>
             </property>
            </item>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>151</width>
              <height>16</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>1.Выберете фигуру</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>370</y>
              <width>151</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Добавить фигуру</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4Punkt">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>240</y>
              <width>191</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>4.Выберете цвет фигуры</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBox_4Punkt">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>270</y>
              <width>151</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Выбирите цвет</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Белый</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Красный</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Синий</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Зелёный</string>
             </property>
            </item>
           </widget>
           <widget class="QLabel" name="label_2Punkt">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>221</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>2. Введите координаты</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>90</y>
              <width>51</width>
              <height>22</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
           <widget class="QLabel" name="label_2p1">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>90</y>
              <width>21</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>90</y>
              <width>51</width>
              <height>22</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
           <widget class="QLabel" name="label_2p2">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>90</y>
              <width>21</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>90</y>
              <width>51</width>
              <height>22</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
           </widget>
           <widget class="QLabel" name="label_2p3">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>90</y>
              <width>21</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Z:</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3Punkt">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>110</y>
              <width>221</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>3.Опциональный параметр</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_3P1">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>150</y>
              <width>51</width>
              <height>22</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
           <widget class="QLabel" name="label_3p2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>180</y>
              <width>131</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Параметр</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_3P2">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>180</y>
              <width>51</width>
              <height>22</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_3P3">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>210</y>
              <width>51</width>
              <height>22</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
           <widget class="QLabel" name="label_3p3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>210</y>
              <width>121</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Параметр</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3p1">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>150</y>
              <width>121</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Параметр</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_5p1">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>340</y>
              <width>51</width>
              <height>22</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
           <widget class="QLabel" name="label_5p2">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>340</y>
              <width>21</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_5Punkt">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>300</y>
              <width>221</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;5. Введите угол поворота &lt;br&gt;относительно оси координат&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_5p2">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>340</y>
              <width>51</width>
              <height>22</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_5p3">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>340</y>
              <width>51</width>
              <height>22</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
           </widget>
           <widget class="QLabel" name="label_5p3">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>340</y>
              <width>21</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_5p1">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>340</y>
              <width>21</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="minimum">
             <double>-180.000000000000000</double>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QCheckBox" name="checkBox">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Вращение с помощью мыши</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="minimum">
             <double>-180.000000000000000</double>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Z:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="minimum">
             <double>-180.000000000000000</double>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
            <property name="stepType">
             <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="GLWidget" name="GLwidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header location="global">glwidget.h</header>
   <container>1</container>
   <slots>
    <signal>signal1()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
